src/Tasks/Common/ConflictResolution/ConflictResolver.cs

```diff
@@ -13,12 +13,16 @@ namespace Microsoft.NET.Build.Tasks.ConflictResolution
     internal delegate void ConflictCallback<T>(T winner, T loser);
 
     //  The conflict resolver finds conflicting items, and if there are any of them it reports the "losing" item via the foundConflict callback
-    internal class ConflictResolver<TConflictItem> where TConflictItem : class, IConflictItem
+    internal class ConflictResolver<TConflictItem> : IDisposable where TConflictItem : class, IConflictItem
     {
         private Dictionary<string, TConflictItem> winningItemsByKey = new Dictionary<string, TConflictItem>();
         private ILog log;
         private PackageRank packageRank;
         private PackageOverrideResolver<TConflictItem> packageOverrideResolver;
+        private Dictionary<string, List<TConflictItem>> unresolvedConflictItems = new Dictionary<string, List<TConflictItem>>(StringComparer.Ordinal);
```

Comment:
Normally, I don't like mixing formatting with other stuff, but here we have code that doesn't follow our guidelines AND moving local to field was part of a fix. Please go ahead and add underscores. :)